src/components/m-accordion/index.scss

```diff
@@ -1,3 +0,0 @@
-.m-accordion {
```

Comment:
please restore, this class is used here:
[`src/components/m-accordion/index.js#L19`](https://github.com/axa-ch/patterns-library/blob/develop/src/components/m-accordion/index.js#L19)